pywinauto/controls/BaseWrapper.py

```diff
@@ -505,16 +540,15 @@ def PressMouseInput(
             key_up = True
     ):
         "Press a mouse button using SendInput"
-        _perform_click_input(
-            self,
-            button,
-            coords,
+        self.ClickInput(
+            button=button,
+            coords=coords,
             button_down=True,
             button_up=False,
             pressed=pressed,
             absolute=absolute,
-            key_down=key_down,
-            key_up=key_up
+#            key_down=key_down,
+#            key_up=key_up
```

Comment:
Please uncomment it. QC warning says `key_up` is unused in the `PressMouseInput` body.